tests/foreman/cli/test_template.py

```diff
@@ -74,6 +80,60 @@ def test_update_template_1(self):
         self.assertEqual(len(result.stderr), 0)
         self.assertEqual(updated_name, result.stdout['name'])
 
+    def test_create_template_with_location(self):
+        """@Test: Check if Template with Location can be created
+
+        @Feature: Template - Create
+
+        @Assert: Template is created and new Location has been assigned
+
+        """
+        try:
+            new_loc = make_location()
+            new_template = make_template({
+                'name': gen_string('alpha', 10),
+                'location-ids': new_loc['id'],
+            })
+        except CLIFactoryError as err:
+            self.fail(err)
+
+        self.assertIn(new_loc['name'], new_template['locations'])
+
+    def test_create_template_locked(self):
+        """@Test: Check that locked Template cannot be created
+
+        @Feature: Template - Create
+
+        @Assert: It is not allowed to create locked Template
+
+        """
+        with self.assertRaises(CLIFactoryError) as ex:
+            make_template({
+                'name': gen_string('alpha', 10),
+                'locked': 'true',
+            })
+        self.assertIn("This template is locked. Please clone it",
+                      ex.exception.message)
```

Comment:
We have exception like:
Failed to create Template with {u'audit-comment': None, u'name': u'wFjJopUYCD', u'file': '/tmp/lZVjPJM5yH', u'type': 'PXELinux', u'organization-ids'"LOCKED": 'true', u'operatingsystem-ids': None, u'location-ids': None} data due to:

  [ERROR 2015-04-03 15:41:59 Exception] This template is locked. Please clone it to a new template to customize.

  Could not create the config template:

    This template is locked. Please clone it to a new template to customize.

Are all these statements going to be localized/translated?